imports/plugins/custom/gdr-landing-page/client/components/CustomLandingPage.js

```diff
@@ -18,16 +21,23 @@ class CustomLandingPage extends Products {
     };
 
     this.getTrendingProducts = this.getTrendingProducts.bind(this);
+    this.initializeTour = this.initializeTour.bind(this);
   }
 
   componentDidMount() {
     this.getTrendingProducts(12);
+    this.initializeTour();
   }
 
+  initializeTour() {
+    setTimeout(() => {
+      onboard.autoDashboardTour(dashboardSteps);
+    }, 2000);
+  }
   getTrendingProducts(limit) {
     Meteor.call("product/getTrendingProducts", limit, (err, res) => {
       if (err) {
-        console.log(err);
+        return err;
```

Comment:
Returning an error here doesn't really do justice to it. Use Meteor's `Logger` with `import { Logger } from "/client/api";` here.